packages/headless/src/api/search/search-api-client.ts

```diff
@@ -83,6 +85,7 @@ export class SearchAPIClient implements FacetSearchAPIClient {
     const response = await PlatformClient.call({
       ...baseSearchRequest(req, 'POST', 'application/json', '/plan'),
       requestParams: pickNonBaseParams(req),
+      requestMetadata: {method: 'plan'},
```

Comment:
So since those name now become part of the API, and means they should be backward compatible, there should be some kind of test that verifies that the contract is never broken.